api/src/main/java/org/infinispan/api/sync/Cache.java

```diff
@@ -0,0 +1,153 @@
+package org.infinispan.api.sync;
+
+import java.util.Iterator;
+import java.util.Map;
+import java.util.function.BiFunction;
+import java.util.function.Function;
+
+import org.infinispan.api.common.KeyValueEntry;
+import org.infinispan.api.common.events.KeyValueListener;
+import org.infinispan.api.configuration.CacheConfiguration;
+
+import io.smallrye.mutiny.Uni;
+
+
+/**
+ *
+ * @since 13.0
+ **/
+public interface Cache<K, V> {
+   /**
+    * Get the value of the Key if such exists
+    *
+    * @param key
+    * @return the value
+    */
+   V get(K key);
```

Comment:
It looks like we are missing a `getAll(Set<K> keys)` or equivalent on all the cache types?

Reply:
Done